src/features/tasks/tasksSlice.js

```diff
@@ -505,6 +589,40 @@ export const updateCameraSerialNumber = (payload) => {
   };
 };
 
+// delete images thunk
+export const deleteImagesTask = (imageIds = [], filters = null, useFilters) => {
```

Comment:
Thoughts on this useFilters flag? Its definitely not necessary but I think its a bit more clear. Maybe can replace with a comment instead.

Reply:
I like it yeah I think it makes it more explicit and clear. Similarly to what I described in [this comment](https://github.com/tnc-ca-geo/animl-frontend/pull/250/files#r1839279938), when flags like that are part of the params I typically pass in objects that have named props so it's even more clear from the point at which you're calling a function what those props do.

E.g. for this one something like `deleteImagesTask({ imageIds: [], filters: filters, useFilters: true })`. That's just personal preference though feel free to leave it as is if you feel otherwise.

Reply:
I ended up deciding against the flag and added a comment while also. bundling the params into an object